index.html

```diff
@@ -345,6 +352,61 @@ <h6>
         </li>
       </ul>
     </footer>
+    <!-- modal begin-->
+         <!-- MODAL SECTION -->
+    <section>
+      <div class="modal" id="modal">
+        <div class="modal-header">
```

Comment:
- [ ] Great job on implementing the modal section, however, a few issues need to be taken care of before we can proceed. Kindly align the buttons in desktop view so that the bottom of the buttons is aligned with the bottom of the page as is in the design.

Figma design
![image](https://user-images.githubusercontent.com/81932478/169661936-2894c39a-1e1f-4584-9419-aff6ba2f2882.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/169661979-88520432-9e2c-4ae6-9487-455ac9ffb03b.png)